conf/cndp_upf_8worker.jsonc

```diff
@@ -0,0 +1,453 @@
+            "qid": 28,
+            "umem": "umem6",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 6 port"
+        },
+        "enp136s0:7": {
+            "pmd": "net_af_xdp",
+            "qid": 29,
+            "umem": "umem7",
+            "region": 1,
+            "busy_poll": true,
+            "busy_budget": 64,
+            "busy_timeout": 20,
+            "description": "Core LAN 7 port"
+        }
+    },
+
+    // (O) Define the lcore groups for each thread to run
+    //     Can be integers or a string for a range of lcores
+    //     e.g. [10], [10-14,16], [10-12, 14-15, 17-18, 20]
+    // Names of a lcore group and its lcores assigned to the group.
+    // The initial group is for the main thread of the application.
+    // The default group is special and is used if a thread if not assigned to a group.
+    "lcore-groups": {
+        "initial": [22],
+        "group0": [25],
+        "group1": [35],
+        "default": ["22-35"]
+    },
+
+    // (O) Set of common options application defined.
+    //     The Key can be any string and value can be boolean, string, array or integer
+    //     An array must contain only a single value type, boolean, integer, string and
+    //     can't be a nested array.
+    //   pkt_api    - (O) Set the type of packet API xskdev or pktdev
+    //   no-metrics - (O) Disable metrics gathering and thread
+    //   no-restapi - (O) Disable RestAPI support
+    //   cli        - (O) Enable/Disable CLI supported
+    //   mode       - (O) Mode type [drop | rx-only], tx-only, [lb | loopback], fwd, acl-strict, acl-permissive
+    //
+    // CNDP BESS Port does not use this section.
+    "options": {
+        "pkt_api": "xskdev",
+        "no-metrics": false,
+        "no-restapi": false,
+        "cli": true,
+        "mode": "drop"
+    },
+
+    // List of threads to start and information for that thread. Application can start
+    // its own threads for any reason and are not required to be configured by this file.
+    //
+    //   Key/Val   - (R) A unique thread name.
+    //                   The format is <type-string>[:<identifier>] the ':' and identifier
+    //                   are optional if all thread names are unique
+    //      group  - (O) The lcore-group this thread belongs to. The
+    //      lports - (O) The list of lports assigned to this thread and cannot share lports.
+    //      description | desc - (O) The description
+    //
+    // CNDP BESS Port does not use this section.
+    "threads": {
+        "main": {
+            "group": "initial",
+            "description": "CLI Thread"
+        },
+        "fwd:0": {
+            "group": "group0",
+            "lports": ["enp134s0:0"],
+            "description": "Thread 0"
+        },
+        "fwd:1": {
+            "group": "group0",
+            "lports": ["enp134s0:1"],
+            "description": "Thread 0"
```

Comment:
Is the `description` correct (thread "0")? I am asking this because the 4worker file has these as ascending numbers 0, 1, 2,...